services/ingestion/etl/cordis/csv/README.md

```diff
@@ -0,0 +1,27 @@
+# CORDIS ETL mapping
+
+Model to compare with is available at: https://ec-europa.github.io/eubfr-data-lake/
+
+| Field                | Target                |
+| -------------------- | --------------------- |
+| rcn                  |                       |
+| id                   | project_id            |
+| acronym              |                       |
+| status               | status                |
+| programme            | sub_programme_name    |
+| topics               |                       |
```

Comment:
This field seems important, but doesn't really fit anywhere. We need to ask Laurent about it.